indexer/db/withdrawal.go

```diff
@@ -17,6 +17,7 @@ type Withdrawal struct {
 	Amount      *big.Int
 	Data        []byte
 	LogIndex    uint
+	BedrockHash *common.Hash
```

Comment:
This is specifically a `StandardBridgeWithdrawal` since it corresponds to a token withdrawal. A general withdrawal is here: https://github.com/ethereum-optimism/optimism/blob/d04d81e8615f4551d438a8e1fd0780749b1c6569/op-chain-ops/crossdomain/withdrawal.go#L15

The scope of this is only token withdrawals?

Reply:
Token and eth withdrawals are the same in the indexer. Eth is represented in the db as a token withdrawal with a special address.

Reply:
The hash is calculated from the message passer WithdrawalInitiated event, which I believe is emitted for both.

Reply:
Got it - the bridge handles eth in the same way, `address(0)` and `address(0x..dead)`

I meant more like arbitrary L2 to L1 messages, which i'm getting the feeling its no

Reply:
Ah got it. Yeah it only supports standard bridge deposits/withdrawals, not arbitrary L1 messages. This is the same as the previous implementation. We only care about the deposits and withdrawals users can see in the gateway UI, so this is OK.

Reply:
There is a wish for the UI to use the lower level contract (apologies for forgetting the name) to do deposits/withdrawals because it's cheaper to use.  But @zainbacchus and others determined that it's not a priority for our OKRs at this time.